client/app/bundles/App/components/site.scss

```diff
@@ -0,0 +1,14 @@
+#root {
+  padding: 20px;
+}
+
+.markdown-body {
```

Comment:
I agree. We should try to move away from using any css that is not in a component. 

The css comes from https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/site.scss

If we need custom css, it is probably best to just make a doc-site component so it is clear to our consumers.